docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,46 @@
 # Contributing to Endless Sky
 
-## Posting issues
+## Issues
+
+### Posting issues
 
 The [issues page](https://github.com/endless-sky/endless-sky/issues) on GitHub is for tracking bugs and feature requests. When posting a new issue, please:
 
-* Check to make sure it's not a duplicate of an existing issue.
+* Check to make sure it is not a duplicate of an existing issue.
 * Create a separate "issue" for each bug you are reporting and each feature you are requesting.
-* Do not use the issues page for things other than bug reports and feature requests. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead.
+* Choose the correct issue template and fill it out to the best of your ability.
+* Do not use the issues page for things other than what match the provided issue templates. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead. Examples of what the discussion pages would be used for include topics that are not looking for a resolution (as the point of an issue is to have it be resolved) or topics that are extremely broad and not befitting of an issue because there is no single resolution to the topic.
+
+If requesting a new feature, first ask yourself: will this make the game more fun or interesting within the existing scope of the game? Remember that this is not a simulator game; while you may find such games fun or interesting, that is not the type of game that Endless Sky is trying to be.
+
+### Closing issues
+
+If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please do not reopen your issue. Please open a new issue and explain the gap that you feel still needs to be addressed.
```

Comment:
```suggestion
If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please do not reopen your issue. Open a new issue and explain the gap that you feel still needs to be addressed.
```

Reply:
I don't think it _really_ matters, but it probably doesn't hurt to err on the side of politeness/friendliness for this file's target audience of newer/less experienced contributors even it repeats please and thank you a bit.